modules/proc_deploy-quay-openshift-operator.adoc

```diff
@@ -761,16 +845,16 @@ SSL certificates can be provided and used instead of having the operator generat
 
 The secret containing custom certificates must define the following keys:
 
-* **ssl.cert**: All of the certificates (root, intermediate, certificate) concatinated into a single file
+* **tls.cert**: All of the certificates (root, intermediate, certificate) concatinated into a single file
 
-* **ssl.key**: Private key as for the SSL certificate
+* **tls.key**: Private key as for the SSL certificate
```

Comment:
Hi, the reason of modifying the key name to `tls.cert` and `tls.key` is for external access handle the TLS verification, right?

Reply:
The Quay UI shows these files as ssl.key and ssl.cert. I'm going to not make the change for now. Please follow up if this needs to be done.

Reply:
Okay. I see what you mean. I'll fix it.